sunpy/instr/goes.py

```diff
@@ -1268,3 +1271,100 @@ def _calc_xraylum(flux, date=None):
     else:
         xraylum = 4 * np.pi * sun.constants.au.to("m")**2 * flux
     return xraylum
+
+def flareclass_to_flux(flareclass):
```

Comment:
Yes, theres a TODO note in the code saying that it misses the quantities.
I dont know whether that one is used within sunpy (tests should tell.... but we don't have 100% coverage). I think some code of hek or helio was using it.  Maybe instead of making it completely go we should be a bit careful (someone may be using it) and mark it as deprecated while internally calling the new one.